README.md

```diff
@@ -15,18 +15,18 @@ cmsenv
 ```tcsh
 cd $CMSSW_BASE/src
 cmsenv
-git clone -b Stop0l git@github.com:susy2015/nanoAOD-tools.git PhysicsTools/NanoAODTools
+git clone -b postprocess_v3.0.0 git@github.com:susy2015/nanoAOD-tools.git PhysicsTools/NanoAODTools
 git clone -b postpro_v2.7 git@github.com:susy2015/NanoSUSY-tools.git PhysicsTools/NanoSUSYTools
-git clone -b Stop0l_NanoAOD_production_V3.0 git@github.com:susy2015/TopTagger.git
+git clone -b Stop0l_NanoAOD_production_V3.1 git@github.com:susy2015/TopTagger.git
 scram b
 cd $CMSSW_BASE/src/TopTagger/TopTagger/test
 ./configure
 make
 cmsenv
 cd $CMSSW_BASE/src/PhysicsTools/NanoSUSYTools/python/processors
-getTaggerCfg.sh -n -t DeepResolved_DeepCSV_GR_nanoAOD_2016_v1.0.2
-getTaggerCfg.sh -n -t DeepResolved_DeepCSV_GR_nanoAOD_2017_v1.0.2
-getTaggerCfg.sh -n -t DeepResolved_DeepCSV_GR_nanoAOD_2018_v1.0.2
+getTaggerCfg.sh -n -t DeepResolved_DeepCSV_GR_nanoAOD_2016_v1.0.3
```

Comment:
Should we add '-o' to this command?

Reply:
shouldn't really be required.  If you already checked it out, then there is no need to download it again, but I don't have any issue with adding "-o"